src/components/EditProfileModal/EditProfileModal.jsx

```diff
@@ -0,0 +1,278 @@
+        )
+
+        if(updateProfileResponse.data.status==="ok")
+        {
+            let responseProfileDetails = updateProfileResponse.data.updatedUserDetails
+            dispatch(updateCurrentProfile(responseProfileDetails))
+            dispatch(updateUserDetails({
+                loggedInUserName: responseProfileDetails.name,
+                loggedInUserEmail: responseProfileDetails.email, 
+                loggedInUserProfile: responseProfileDetails.profilePicSrc
+            }))
+            setShowEditProfileModal(false)
+        }
+    }
+    
+    return (
+        <div className="edit-profile-modal-container">
+            
+            <div className="edit-profile-modal">
+                <div className="edit-profile-modal-header">
+                    <h3>Edit Profile</h3>
+                    <div 
+                        className="close-edit-modal-btn"
+                        onClick={()=>setShowEditProfileModal(false)}
+                    >
+                        <CgClose className="close-edit-modal-btn-icon"/>
+                    </div>
+                </div>
+                <hr></hr>
+
+                <input
+                    type="file"
+                    style={{display:"none"}}
+                    ref={addProfilePicFileInput}
+                    accept="image/jpeg,image/png,image/webp,
+                    image/gif"
+                    onChange={event=>{
+                      const file = event.target.files[0]
+                      uploadProfileImage(file)
+                    }}
+                />
+
+                <div className="edit-user-profilepic-container">
+                    <div className="edit-profile-detail-header">
+                        <h3>Profile picture</h3>
+                        <p 
+                            className="profile-edit-text"
+                            onClick={addNewProfilePicFile} 
+                        >
+                            Edit
+                        </p>
+                    </div>
+                    {
+                        selectedProfileFileUrl!=="" ?
+                        (
+                            <Image
+                            className="edit-details-profile-pic"
+                            cloudName="dddfc84ni"
+                            publicId={selectedProfileFileUrl}
+                            />
+                        ): (
+                            <img 
+                                className="edit-details-profile-pic" 
+                                src={userProfilePic!==""?userProfilePic:"https://enztron-dev-branch.netlify.app/Icons-and-Images/Avatars/blue-illustration-avatar.svg"} 
+                                alt="user profile pic"
+                            >
+                            </img>
+                        )
+                    }
+                </div>
+
+                <input
+                    type="file"
+                    style={{display:"none"}}
```

Comment:
try to avoid inline styles but if this bit was unavoidable then its alright

Reply:
Done, added separate class